pkg/conf/conf.go

```diff
@@ -215,7 +230,7 @@ func GetHarvestHomePath() string {
 func GetHarvestLogPath() string {
 	var logPath string
 	if logPath = os.Getenv("HARVEST_LOGS"); logPath == "" {
-		logPath = "/var/log/harvest/"
+		logPath = "/Users/hardikl/Documents/HarvestPublic/harvest/log/harvest/"
```

Comment:
Whoops. Make sure you change this back. I assume you changed this because you're on a Mac? If so, ping me, and I'll show you a better solution using the env var above

Reply:
Yups.
Reverting it.